### ISSUE TITLE
[chr15:31496985-31637314][DUP][SVLEN=140329bp]

### ISSUE THREAD

**Issue by jzook** [jzook/testHG008curation#255]
`chr15	31496985	Minda_43	N	<DUP>	.	PASS	SVLEN=140329;SVTYPE=DUP;SUPP_VEC=PB_severus_DUP2179,PB_r_122,ONT_severus_DUP2435,ONT_r_93,ILL_MantaDUP:TANDEM:217536:0:1:0:0:0,ILL_2864072751:2,ILL_gridss233bf_1208h,PB_ID_20429_2,ONT_ID_72288_2,ONT_Sniffles2.DUP.E0ME
`

https://v2.genomeribbon.com/?session=https://42basepairs.com/download/s3/giab-data/ribbon-json/ribbon-hg008-cov-bedpe.json&locus=chr15:31496985#ribbon

**Comment by aquaskyline** [comment]
Correct tandem duplication. Breakpoints are exact and match Dragen results. Likely clonal, reads supporting the dup are all in the unphased group in Pacbio.

![Screenshot 2024-10-13 at 21 26 28](https://github.com/user-attachments/assets/c9c62751-9c21-45a2-bfdf-d8ff700607f4)
![Screenshot 2024-10-13 at 21 29 18](https://github.com/user-attachments/assets/9ea97dd8-b4c0-4f06-94ec-c164587a0be8)
![Screenshot 2024-10-13 at 21 25 53](https://github.com/user-attachments/assets/5cc6424a-5602-4aeb-8736-91652fc74dbb)

Attention needed that there is a deletion on a long tandem repeat within the duplication.

![Screenshot 2024-10-13 at 21 27 57](https://github.com/user-attachments/assets/e14b4efa-d21e-4e15-92fd-b76978b68a19)

**Comment by jzook** [comment]
I agree this looks like a clonal tandem dup. I think we can ignore the germline deletion inside the tandem dup, though this technically does affect the length of the duplication

**Comment by jzook** [comment]
Unfortunately the HG008Nv6.3 normal assembly has a gap inside this duplication that makes it complicated, perhaps because of an error when curating the assembly graph, but can still see the breakpoints outside the gap

<img width="1253" height="938" alt="Image" src="https://github.com/user-attachments/assets/43dd95b7-c589-4f12-9e6f-5d6c0a14c7e9" />
